src/modules/fundingManager/oracle/interfaces/IOraclePrice_v1.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+/**
+ * @title   Oracle Price Interface
+ *
+ * @notice  Interface for oracle price feed calculations that provides a standardized
+ *          way to query token prices for both issuance and redemption operations.
+ *
+ * @dev     Designed to facilitate various oracle price implementations. Each
+ *          implementation must provide methods to get current prices for both
+ *          buying (issuance) and selling (redemption) operations.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @author  Zealynx Security
+ */
+interface IOraclePrice_v1 {
+    //--------------------------------------------------------------------------
+    // Errors
+
+    /// @notice Thrown when price returned is zero
+    error OraclePrice__ZeroPrice();
+
+    //--------------------------------------------------------------------------
+    // External Functions
+
+    /// @notice Gets current price for token issuance
+    /// @return price_ Current price for buying tokens
```

Comment:
We might want to add information about in which token the price is denominated here and with the redemption one as well. This would help future devs to implement it knowing what to return. Wdyt?

Reply:
Could maybe also be added in the interface natspec

Reply:
I hope now is clearer. I've added extra info